server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -56,7 +57,8 @@
 import io.crate.testing.UseRandomizedOptimizerRules;
 import io.crate.testing.UseRandomizedSchema;
 
-@IntegTestCase.ClusterScope(numDataNodes = 2)
+@TestLogging("io.crate.lucene:TRACE,org.elasticsearch.index.mapper:TRACE")
```

Comment:
can these be reverted?

Reply:
Sure, my fault, missed to remove these debug traces. Thanks!